typescript/sdk/src/consts/multisigIsm.ts

```diff
@@ -2,24 +2,34 @@ import { MultisigConfig } from '../ism/types';
 import { ChainMap } from '../types';
 
 export const defaultMultisigConfigs: ChainMap<MultisigConfig> = {
-  // ----------------- Mainnets -----------------
-  celo: {
+  alfajores: {
     threshold: 2,
     validators: [
-      '0x63478422679303c3e4fc611b771fa4a707ef7f4a',
-      '0x622e43baf06ad808ca8399360d9a2d9a1a12688b', // dsrv
-      '0xf2c1e3888eb618f1f1a071ef3618f134715a9a49', // everstake
+      '0x2233a5ce12f814bd64c9cdd73410bb8693124d40',
```

Comment:
Can we exclude these objects from the lint rule? Changing the order will I believe cause re-deploy of ISMs which we should avoid (+ i think break the ci fork tests rn)

Reply:
Actually generally seems weird for a lint rule to enforce ordering of string constants in an array?

Reply:
The diff is misleading you here. The eslint fixer didn't reorder any values in the lists here. It reordered the objects at the chain name level.
The fork tests have been broken for a few days at least.